osuosl/timesync-frontend-flask
# Create/Edit/View Activity Pages

fixes issue #30 
## Changes in this PR.


- [X] Admin-only create/update activity pages
- [X] Admin-only view activities page
## Testing this PR.


1. Run `nosetests`
2. Run the app and login as `admin` with the password `timesync-staging`.
3. Navigate to the `Admin` page
4. Click on `Create Activity` to create an activity
5. Go back, then go to `View Activities` and click on the link to edit the new activity.
6. Go back to `View Activities` and see the updated activity (If you don't see it, try refreshing the page).
### Expected Output.



```
$ nosetests
...........................................
----------------------------------------------------------------------
Ran 43 tests in 0.666s

OK
```

@osuosl/devs
